### DISCUSSION TITLE
Feature request: import icons into kits

### DISCUSSION THREAD

**Discussion by bennyb** [FortAwesome/Font-Awesome#20921] [Feature Requests]
### Is your feature request related to a problem?

When creating new kits with custom subsets would be good to import a list of icons without having to manually select each one.

### Feature description

Be good to list in say a CSV font names and style which can then be imported into a kit.  Also be good to have a way of duplicating a kit so all the icons don't have to be manually selected for kits with similar usage.  

### Feature request checklist

- [X] The title starts with "Feature request: " and is followed by the requested feature description
- [X] This is a single feature
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by kokoshneta** [comment]
I would argue that this is an **absolute necessity**, not just something that would be nice to have.

Creating a kit with 300 selected icons is an exercise in frustration, particularly when you already have all the icon names in a neat list in a file.
